.gitignore

```diff
@@ -59,3 +59,4 @@ prof/**
 docs/evm.*rst
 venv*
 .eggs/
+tests/rpc/mainnet_tester_chain.db/*
```

Comment:
I'm a big fan of relegating things to a `fixtures/` dir if you're into that sort of thing.